qgroundcontrol.qrc

```diff
@@ -17,6 +17,8 @@
         <file alias="ROIIndicator.qml">src/ui/toolbar/ROIIndicator.qml</file>
         <file alias="TelemetryRSSIIndicator.qml">src/ui/toolbar/TelemetryRSSIIndicator.qml</file>
         <file alias="VTOLModeIndicator.qml">src/ui/toolbar/VTOLModeIndicator.qml</file>
+        <file alias="FuelIndicator.qml">src/ui/toolbar/FuelIndicator.qml</file>
```

Comment:
These changes are not included in the documentation

Reply:
```suggestion
        <file alias="FuelIndicator.qml">src/ui/toolbar/FuelIndicator.qml</file>
```
fuel indicator is not implemented separately imstead uses battery status indicator.
this `FuelIndicator.qml` is not required.